### ISSUE TITLE
Compilation error in wasm

### ISSUE THREAD

**Issue by ineiti** [lrlna/femme#40]
It seems that compiling in wasm is not in the test suite, as it fails. I create a PR here: #39

**Comment by usagi** [comment]
@lrlna the author,

I see #39, the patch was merged. But why you would not publish a new release to crates.io include the patch yet? This patch is very critical if femme supports wasm. Because, this problem be occur in very basic usage as below:

```sh
## Repro.
cargo new test-femme --lib
cd test-femme
cargo add femme log wasm-bindgen 
echo '[lib]' >> Cargo.toml
echo 'crate-type = ["cdylib"]' >> Cargo.toml
wasm-pack build
```

![image](https://user-images.githubusercontent.com/115972/167346611-5074b966-c111-457f-860e-da75a6cf086c.png)

Could you publish a new patch release, please?

**Comment by ineiti** [comment]
In the meantime you can reference the github repo directly in `Cargo.toml`:

```
femme = { git = "https://github.com/lrlna/femme" }
```

**Comment by lrlna** [comment]
published in [`2.2.1`](https://crates.io/crates/femme)
